core/src/main/java/com/sequenceiq/cloudbreak/service/LoadBalancerConfigService.java

```diff
@@ -405,9 +491,9 @@ private LoadBalancer createLoadBalancerIfNotExists(Set<LoadBalancer> loadBalance
         return loadBalancer;
     }
 
-    private void setupKnoxLoadBalancer(LoadBalancer loadBalancer, TargetGroup knoxTargetGroup) {
-        loadBalancer.addTargetGroup(knoxTargetGroup);
-        knoxTargetGroup.addLoadBalancer(loadBalancer);
+    private void setupLoadBalancerWithTargetGroup(LoadBalancer loadBalancer, TargetGroup targetGroup) {
+        loadBalancer.addTargetGroup(targetGroup);
+        targetGroup.addLoadBalancer(loadBalancer);
```

Comment:
**praise:** Thanks for genericizing the parameter name `knoxTargetGroup` -> `targetGroup`.